src/hazelcore/dynamics/Dynamics.re

```diff
@@ -720,6 +722,8 @@ module Exp = {
     | BinBoolOp(_, _, _)
     | BinIntOp(_, _, _)
     | BinFloatOp(_, _, _)
+    | AssertLit => Indet
+    | FailedAssert => Indet
```

Comment:
FYI this is a bug. Fixing in my merge.

Reply:
I think this should say `AssertLit` is `Indet` but `FailedAssert` is the same as `Triv`, since any non-indet assert evaluates to unit. @cyrus- can you confirm?

Reply:
Sorry this is not the right version the up to date pull request is Assert numbering ui. This one just have the very simple assert implementation. Assert numbering ui has the feature about assert mapping ad UI implementation.

Reply:
doh ok